windowsforms/FlexGridShowcaseDemo/cs/FlexGridShowcaseDemo.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+  
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+    <ApplicationIcon>App.ico</ApplicationIcon>
+    <Company>GrapeCity, Inc.</Company>
+    <Copyright>(c) GrapeCity, Inc..  All rights reserved.</Copyright>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
Why this is false?

Reply:
I've removed EnableDefaultCompileItems, EnableDefaultEmbeddedResourceItems, EnableDefaultNoneItems properties and explicit listing of files - 01ddeab6a7ae800fd7e9236ab08e01416af611cb.